contracts/interfaces/IbcMiddleware.sol

```diff
@@ -247,3 +250,29 @@ contract IbcMwUser is Ownable {
         authorizedMws[address(middleware)] = true;
     }
 }
+
+abstract contract IbcUniversalPacketReceiverBase is IbcMwUser, IbcUniversalPacketReceiver {
```

Comment:
For clarity, should we move this base contract along with `IbcUniversalPacketReceiver` to a new file `IbcUniversalChannelReceiver.sol` ?

Reply:
I agree we should do this; In general the example contracts/interfaces need some cleanup/clarifications. There's an open issue on the kanban to tack this. 

I think we should do any organization/clarity changes in a separate PR to separate out audit-specific fixes/docs from general clarity ones so I think we can do this change in another PR?